### ISSUE TITLE
Integrations for #accessibility-events

### ISSUE THREAD

**Issue by akirk** [Automattic/matrix-bots-wporg#13]
Accordingt to our records, the `#accessibility-events` Slack channel on Making WordPress does not have any integrations that need to be migrated from Slack to Matrix.
